train.py

```diff
@@ -1,52 +1,67 @@
+from calendar import c
+from locale import currency
 from PIL import Image
 import os
 from network import *
 from dataset import *
+from torch.utils.tensorboard import SummaryWriter
+
 
 device = 'cpu'
-learn_rate = 3e-4
+learn_rate = 3e-5
 z_dim = 32
-img_dim = 28*28
+img_dim = 28
 batch_size = 32
-num_epochs = 1
+num_epochs = 50
 
-df = pd.read_csv('english.csv')
-train_dataset = Data('english.csv')
+train_dataset = Data('english.csv', img_dim)
 train_dataloader = DataLoader(train_dataset, batch_size, shuffle = True)
+gan = GAN(img_dim*img_dim, z_dim).to(device)
 
-print(len(df))
+writer = SummaryWriter()
+gan.load_state_dict(torch.load('model_epoch_9.pt'))
 
-gen = Generator(img_dim, z_dim).to(device)
-disc = Discriminator(z_dim).to(device)
-z = torch.randn(batch_size, z_dim).to(device)
-gen_opt = torch.optim.Adam(gen.parameters(), lr=learn_rate)
-disc_opt = torch.optim.Adam(disc.parameters(), lr=learn_rate)
-criterion = nn.BCELoss()
+for epoch in range(10, num_epochs):
+    batch_idx = 0
```

Comment:
mozes da se stavi enumerate oko for loop-a vise je pythonic nacin kucanja

Reply:
vazi